services-turf/src/main/java/com/mapbox/turf/TurfAssertions.java

```diff
@@ -69,7 +69,7 @@ public static void featureOf(Feature feature, String type, String name) {
       throw new TurfException(String.format(
         "Invalid input to %s, Feature with geometry required", name));
     }
-    if (feature.geometry() == null || !feature.geometry().type().equals(type)) {
+    if (!feature.geometry().type().equals(type)) {
```

Comment:
Nice! `geometry()` has it's own null check so we do not need the explicit null check anymore.